src/Imgur.API/Endpoints/Impl/EndpointBase.cs

```diff
@@ -69,10 +71,10 @@ internal EndpointBase(IApiClient apiClient, HttpClient httpClient)
                     : $"Client-ID {apiClient.ClientId}");
 
             //Add Mashape Authentication header
-            if (apiClient is IMashapeClient mashapeClient)
+            if (apiClient is IMashapeClient)
```

Comment:
My code is a C# 7 feature called Pattern Matching. I'm not sure why you're trying to revert it.
https://csharp.christiannagel.com/2017/02/15/patternmatching/